src/utils/resolveMessage.ts

```diff
@@ -0,0 +1,72 @@
+import { Colors, EmbedBuilder, Message, ThreadChannel } from "discord.js";
```

Comment:
import types as types!

```suggestion
import { Colors, EmbedBuilder, type Message, ThreadChannel } from "discord.js";
```